docs/blog/20231004-Daeraxa-OctoberUpdate.md

```diff
@@ -0,0 +1,166 @@
+- So instead, a request is made and the package is forked to the Pulsar Cooperative organization where bug fixes and new features can be added by _any_ community member and so long as those fixes and features do not fundamentally change the package, introduce malicious code, or otherwise break functionality, they will be accepted and a new release published automatically.
+
+We hope to introduce a place where people can cooperatively work to make the Pulsar ecosystem better without having to worry about violating the strict vision for a package and not having their fix even accepted. It is also a place where people can submit fixes and make their favourite packages functional again without necessarily having to take on full responsibility for the ongoing maintenance.
+
+This is not a "Pulsar team" project in the sense that members of the main organization are going to be providing bug fixes or updates in response to logged issues (of course they are free too, just like any other community member). This is an entirely community-led endeavour and we hope everyone will be encouraged to get involved!
+
+This isn't ready _just_ yet, but look out for an update right here on the blog (or any of our other community areas) once we are fully live and ready!
+
+## Modern Tree-sitter blog posts
+
+<img src="./assets/tree-sitter.png" height="150" />[^1]
+
+If you hadn't already seen or read them, [@savetheclocktower] has been writing a series of blog posts that go into detail about all the work that has been going on in Pulsar to support a modern implementation of [Tree-sitter](https://tree-sitter.github.io/tree-sitter/). Parts [one](https://pulsar-edit.dev/blog/20230925-savetheclocktower-modern-tree-sitter-part-1.html) and [two](https://pulsar-edit.dev/blog/20230927-savetheclocktower-modern-tree-sitter-part-2.html) are available right now and keep an eye on the Pulsar blog for more updates!
+
+## Converting PPM's code from callbacks to async
+
+<img src="./assets/development.png" height="150" />
+
+Community member [@Nemokosch/@twocolours]  has been spearheading the conversion of our [PPM codebase](https://github.com/pulsar-edit/ppm) to upgrade old JavaScript callback style to modern ES6 `async/await`.
+
+The PPM repo, while modern and accepted when written, is now mostly viewed as being outdated due to the pervasiveness of deeply nested callbacks. In some instances haveing nesting six levels deep, and as anyone who's been programming in JavaScript prior to ES6 knows, this can be cumbersome, difficult, and frustrating to work with. That is exactly why the effort has been made to convert the entire codebase into a shallow nested, early return patterned, modern ES6 source utilizing async/await properly. In hopes that a greatly simplified codebase can pave the way for easier modifications in the future.
```

Comment:
```suggestion
The PPM repo, while modern and accepted when written, is now mostly viewed as being outdated due to the pervasiveness of deeply nested callbacks. In some instances having nesting six levels deep, and as anyone who's been programming in JavaScript prior to ES6 knows, this can be cumbersome, difficult, and frustrating to work with. That is exactly why the effort has been made to convert the entire codebase into a shallow nested, early return patterned, modern ES6 source utilizing async/await properly. In hopes that a greatly simplified codebase can pave the way for easier modifications in the future.
```